app/questions/services/mailchimp.py

```diff
@@ -0,0 +1,59 @@
+from mailchimp3 import MailChimp
+
+from django.conf import settings
+from django.utils import timezone
+
+from questions.models import Submission
+
+
```

Comment:
I know this is a little pedantic but in future can you please use normal punctuation for comments
eg
> Master function finding clients and sending reminder emails to them.
or
> Master function finding clients and sending reminder emails to them

Reply:
in addition can you please format the code with `black`, happy to provide instructions on how to do that if you need help with it

Reply:
done